composer.json

```diff
@@ -12,10 +12,17 @@
     "support": {
         "irc": "irc://irc.freenode.net/phergie"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/WyriHaximus-labs/phergie-irc-client-react"
+        }
+    ],
     "require": {
+        "phergie/phergie-irc-client-react": "dev-reactphp-1.0 as 3.1.0",
```

Comment:
This and the `repositories` section above are require until https://github.com/phergie/phergie-irc-client-react/pull/65 has been finished, merged, and tagged.